### PR TITLE
OT-1731 fb camera input

### PR DESCRIPTION
**Pull request by the7winds**
Here:
* use CameraInput in FaceBuilder's constructor
* drop deprecated methods like `update_projection`, `update_image_size`
* merge `place_camera` and `place_cameraobj` into a single method

### CODE REVIEW

**Comment by the7winds** [comment]
@SergeyKrivohatskiy

File: keentools_facebuilder/camera_input.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        return settings.get_camera(settings.current_headnum, frame - 1)
+
+    def projection(self, frame):
+        return self._camera_at(frame).get_projection_matrix()
+
+    def view(self, frame):
+        return np.eye(4)
+
+    def image_size(self, frame):
+        return self._camera_at(frame).get_image_size()
```
Comment (SergeyKrivohatskiy) [inline keentools_facebuilder/camera_input.py:21]: I see get_oriented_image_size was in update_camera_projection. Is get_image_size ok here?

**Review by SergeyKrivohatskiy — APPROVED** [review]
I cannot check the code is correct, but is looks good. Better than it was. Please check everything works as it should. FL estimation for example.

File: keentools_facebuilder/camera_input.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+from . import FBCameraItem
+from .settings import get_main_settings, FBSceneSettings
+
+
+class FaceBuilderCameraInput(FaceBuilderCameraInputI):
+    @staticmethod
+    def _camera_at(frame) -> FBCameraItem:
+        settings: FBSceneSettings = get_main_settings()
+        return settings.get_camera(settings.current_headnum, frame - 1)
```
Comment (amilovsky) [inline keentools_facebuilder/camera_input.py:12]: What is the "frame" variable responsible for? Is this the camera number in settings or keyframe number? Anyway (frame - 1) looks suspicious. Camera keyframes (keyframe_id) can be in [1, 5, 6, 9] by ex. I think it needs a search camera by keyframe_id

File: keentools_facebuilder/camera_input.py (hunk 1 above)
Comment (amilovsky) [reply keentools_facebuilder/camera_input.py:21]: Image orientation should possibly affect the returned size. Not sure about the goal of this image_size

File: keentools_facebuilder/fbloader.py [hunk 3]
```diff
@@ -41,7 +40,9 @@ def viewport(cls):
 
     @classmethod
     def new_builder(cls):
-        cls._builder_instance = pkt.module().FaceBuilder()
+        from .camera_input import FaceBuilderCameraInput
+        cls._camera_input = FaceBuilderCameraInput()
```
Comment (amilovsky) [inline keentools_facebuilder/fbloader.py:44]: It might be better to move this import up, not in method

**Pull request merged**